ServerCore/Pages/Teams/Apply.cshtml

```diff
@@ -0,0 +1,23 @@
+﻿@page  "/{eventId}/{eventRole}/Teams/Apply"
+@model ServerCore.Pages.Teams.ApplyModel
+
+@{
+    ViewData["Title"] = "Apply";
+}
+
+<h2>Apply to a Team</h2>
+
+<hr />
+<div class="row">
+    <div class="col-md-4">
+        You have applied to <strong>@{@Model.Team.Name}</strong>. Once they accept you, you will be on the team.
```

Comment:
I would include a couple more points here:

1. If you had previously applied to a different team, that application has been removed as you can only apply to one team at a time.
2. Your team will not be notified of your application, and it is recommended that you follow up with email to close the loop.